code/modules/admin/verbs/adminhelp.dm

```diff
@@ -188,14 +198,19 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 
 	statclick = new(null, src)
 	_interactions = list()
+	if(mentor_status)
+		is_mentor_ticket = TRUE
 
 	if(is_bwoink)
 		AddInteraction("<span class='notice'>[key_name_admin(usr)] PM'd [LinkedReplyName()]</span>")
 		message_admins("<span class='notice'>Ticket [TicketHref("#[id]")] created</span>")
+
 	else
 		MessageNoRecipient(msg)
-
-		SSredbot.send_discord_message("admin", "Ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord
+		if(is_mentor_ticket)
+			SSredbot.send_discord_message("admin", "Mentor ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord
+		else
+			SSredbot.send_discord_message("admin", "Admin Ticket #[id] created by [usr.ckey] ([usr.real_name]): [name]", "ticket") //send the ticket to discord

```

Comment:
@crossedfall mind confirmed this wont break anything with the bot?

Reply:
I confirm it wont break anything. That being said, I have to ask if you actually want mentor tickets to be sent to discord in the same way as ahelps.

Reply:
I think it's valuable for mentors to see if there are any gameplay questions they can easily answer

Reply:
Changes made to the bot. See my new comment with suggested changes.